WebDevStudios/nextjs-wordpress-starter
# Feature/next image

Closes #241 

### Link
https://nextjs-wordpress-starter-git-feature-next-image-webdevstudios.vercel.app/

### Description
This PR introduces support for the  next/image [component](https://nextjs.org/docs/api-reference/next/image).

### Screenshot
![screenshot](https://dl.dropbox.com/s/u744dkpobxnnvd6/Screen%20Shot%202021-02-26%20at%2012.25.21.png?dl=0)

### Verification
- Visit the [URL above and click around](https://nextjs-wordpress-starter-git-feature-next-image-webdevstudios.vercel.app/) (note that one of the blog posts has missing images, but that's unrelated to this PR)
- Run a Lighthouse test to verify properly sized/next-gen image format
